packages/appkit/src/client.ts

```diff
@@ -648,7 +648,7 @@ export class AppKit {
   }
 
   public async disconnect() {
-    await ChainController.disconnect()
+    await ConnectionController.disconnect()
```

Comment:
😮

Reply:
It should have been ConnectionController from the beginning

Reply:
🙈